kopf/reactor/loading.py

```diff
@@ -14,17 +14,20 @@
 Multiple files/modules can be specified. They will be loaded in the order.
 """
 
+import sys
 import importlib
 import importlib.util
 import os.path
 
 
+
 def preload(paths, modules):
     """
     Ensure the handlers are registered by loading/importing the files/modules.
     """
 
     for path in paths:
+        sys.path.append(os.path.abspath(os.path.curdir))
```

Comment:
Since Kopf promises to mimic Python here, it is better to prepend the path here instead of appending.

Also, Python adds not the current directory, but the directory of the script being executed. Can be tested this way:

```bash
$ mkdir dir1
$ echo 'import sys; print(repr(sys.path))' > dir1/file1.py
$ python dir1/file1.py
['/Users/svasilyev/dir1', .....]
```

```suggestion
        sys.path.insert(0, os.path.abspath(os.path.dirname(path)))
```

Kopf has slightly enhanced semantics here with multiple scripts in one command line: `kopf run dir1/script1.py dir2/script2.py ...`. The dirs will be prepended in reverse order. But I don't think it is a big issue for now (can be fixed later).

The resulting `sys.path` will be `['.../proj/dir2', '.../proj/dir1', '/other/system/dirs']`. The relative modules/packages will be importable then.